crates/goose-server/src/commands/mcp.rs

```diff
@@ -16,8 +17,19 @@ pub async fn run(name: &str) -> Result<()> {
     }
 
     tracing::info!("Starting MCP server");
+
+    if name == "developer" {
```

Comment:
??? why is this one different now? this seems a lot more complicated than the old router?